libraries/Microsoft.Bot.Streaming/Payloads/Disassemblers/CancelDisassembler.cs

```diff
@@ -22,7 +22,9 @@ public CancelDisassembler(IPayloadSender sender, Guid id, char type)
 
         private char Type { get; set; }
 
+#pragma warning disable UseAsyncSuffix // Use Async suffix  (we can't change this without breaking binary compat)
         public Task Disassemble()
+#pragma warning restore UseAsyncSuffix // Use Async suffix
```

Comment:
>UseAsyncSuffix [](start = 24, length = 14)

cancelationToken

Reply:
It is not needed by any downstream call, we can add an optional parameter in the feuture when is required.